pkg/webhooks/regularuser/regularuser.go

```diff
@@ -225,6 +241,43 @@ func isCustomDomainAuthorized(request admissionctl.Request) bool {
 		request.Kind.Group == customDomainGroup)
 }
 
+// isNetNamespaceAuthorized check if request is authorized for NetNamespace CR
+func isNetNamespaceAuthorized(s *RegularuserWebhook, request admissionctl.Request) bool {
+	return ((utils.SliceContains("cluster-admins", request.UserInfo.Groups) ||
+		utils.SliceContains("dedicated-admins", request.UserInfo.Groups)) &&
+		request.Kind.Kind == netNamespaceKind &&
+		request.Kind.Group == netNamespaceGroup &&
+		isNetNamespaceValid(s, request))
+}
+
+// isNetNamespaceValid check if the NetNamespace is valid
+func isNetNamespaceValid(s *RegularuserWebhook, request admissionctl.Request) bool {
+	// Decode object into a NetNamespace object
+	decoder, err := admissionctl.NewDecoder(&s.s)
+	if err != nil {
+		return false
+	}
+	netNamespace := &networkv1.NetNamespace{}
+	if len(request.Object.Raw) == 0 {
+		return false
+	} else {
+		err := decoder.Decode(request, netNamespace)
+		if err != nil {
+			return false
+		}
+	}
+	// Check if name is empty
+	if len(netNamespace.Name) == 0 {
+		return false
+	}
```

Comment:
This is not a very useful comment because we can read the following line and understand that it's a check to see if the length of the name is 0. A better comment would be to explain why this is important.

Additionally, an object's `Name` [is guaranteed to be non-empty](https://github.com/kubernetes/community/blob/b345baf47f2b8a55acf9344f0f0412f49b2a3179/contributors/design-proposals/architecture/identifiers.md), so this check can also be removed.
```suggestion

```

Reply:
Done.